LiteCore/Support/c4ExceptionUtils.hh

```diff
@@ -43,11 +43,23 @@ namespace c4Internal {
     /** Macro to substitute for a regular 'catch' block, that saves any exception in OUTERR. */
     #define catchError(OUTERR) \
         catch (const std::exception &x) { \
+            C4WarnError("Recording exception caught in %s: %s", __func__, x.what()); \
             c4Internal::recordException(x, OUTERR); \
         }
 
     #define catchExceptions() \
-        catch (const std::exception &) { }
+        catch (const std::exception &x) { \
+            C4WarnError("Exception caught in %s: %s", __func__, x.what()); \
+        }
+
+    #define catchTripleDot() \
```

Comment:
I don't think this is needed. There are no exceptions in our code or the std lib that don't inherit from `std::exception`.

Reply:
I guess it would be better to just have the fatal crash trace on by default?